pkg/controller/replicaset/replica_set.go

```diff
@@ -115,6 +115,22 @@ type ReplicaSetController struct {
 
 	// Controllers that need to be synced
 	queue workqueue.TypedRateLimitingInterface[string]
+
+	// Features that can be set in accordance with the controller type (GVK).
+	// These do not have to correspond to FeatureGates, or their stability, nor do they have to undergo graduation.
+	controllerFeatures ReplicaSetControllerFeatures
```

Comment:
```suggestion
        // controller specific feature toggles. see ReplicaSetControllerFeatures for details.
        controllerFeatures ReplicaSetControllerFeatures
```

[imho] from a reader’s standpoint the comments feel a bit duplicative,how about keeping just a short reference on the field?

Reply:
Good idea, updated!